_pages/cfp.md

```diff
@@ -5,7 +5,8 @@ subtitle: Call for Proposals
 permalink: /cfp/
 ---
 
-Our CFP will open on August 14 2017 and close on September 15 2017! 
+Our call for proposals (CFP) for 15 minute lightning talks will open on August 14 2017 and close on September 15 2017!
```

Comment:
I'd rather keep it and be suuuper explicit about what the shortform is for (have them right beside each other) since it confuses people so much